packages/playground/public/loader/loader.js

```diff
@@ -47,22 +47,27 @@ refreshBtn &&
   );
 
 window.$$appLoader = () => {
-  refreshBtn && refreshBtn.classList.remove("active");
-  t1 && clearTimeout(t1);
-  t2 && clearTimeout(t2);
-
-  refreshBtn && refreshBtn.classList.remove("active");
-  loader && loader.classList.remove("active");
-  if (msgElement) {
-    msgElement.textContent = "Dashboard loaded. Welcome!";
-  }
-
-  setTimeout(() => {
-    if (appLoaderContainer) {
-      appLoaderContainer.classList.remove("active");
-      setTimeout(() => appLoaderContainer.remove(), appLoaderContainerTime);
+  return new Promise(res => {
+    refreshBtn && refreshBtn.classList.remove("active");
+    t1 && clearTimeout(t1);
+    t2 && clearTimeout(t2);
+
+    refreshBtn && refreshBtn.classList.remove("active");
```

Comment:
Do you remove this line twice? 
```ts
    refreshBtn && refreshBtn.classList.remove("active");
```

Reply:
this is the old implementation I just made it return promise. I don't  know the idea behind it, honestly

Reply:
Please remove the repeated code if not needed